docs/learn/bitesize-invaders.md

```diff
@@ -0,0 +1,309 @@
+///     CheckForAllPlayersReady
+///     Checks to see if all players are ready, and if so launches the game
+/// </summary>
+private void CheckForAllPlayersReady()
+{
+    if (m_AllPlayersInLobby)
+    {
+        var allPlayersAreReady = true;
+        foreach (var clientLobbyStatus in m_ClientsInLobby)
+            if (!clientLobbyStatus.Value)
+
+                //If some clients are still loading into the lobby scene then this is false
+                allPlayersAreReady = false;
+
+        //Only if all players are ready
+        if (allPlayersAreReady)
+        {
+            //Remove our client connected callback
+            NetworkManager.Singleton.OnClientConnectedCallback -= OnClientConnectedCallback;
+
+            //Remove our scene loaded callback
+            SceneTransitionHandler.sceneTransitionHandler.OnClientLoadedScene -= ClientLoadedScene;
+
+            //Transition to the ingame scene
+            SceneTransitionHandler.sceneTransitionHandler.SwitchScene(m_InGameSceneName);
+        }
+    }
+}
+```
+
+<!-- ```csharp reference
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/LobbyControl.cs#L158-L186
+``` -->
+
+## Unconventional Networked Movement
+
+Invaders has an easy movement type - moving only on one (horizontal) axis - which allows you to only modify the transform client-side. Modify *[PlayerControl.cs](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/PlayerControl.cs)* in the `InGameUpdate` function with the help of a [`NetworkTransform`](../components/networktransform.md) that is attached directly to the Player Game Object and automatically syncs up the Transform with the other clients. It will smooth out by interpolating or extrapolating for all of them.
+
+```csharp
+private void InGameUpdate()
+{
+    if (!IsLocalPlayer || !IsOwner || !m_HasGameStarted) return;
+    if (!m_IsAlive) return;
+
+    var deltaX = 0;
+    if (Input.GetKey(KeyCode.LeftArrow)) deltaX -= 1;
+    if (Input.GetKey(KeyCode.RightArrow)) deltaX += 1;
+
+    if (deltaX != 0)
+    {
+        var newMovement = new Vector3(deltaX, 0, 0);
+        transform.position = Vector3.MoveTowards(transform.position, transform.position + newMovement, m_MoveSpeed * Time.deltaTime);
+    }
+
+    if (Input.GetKey(KeyCode.Space)) ShootServerRPC();
+}
+```
+
+<!-- ```csharp reference
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/PlayerControl.cs#L157-L173
+``` -->
+
+## Shared Start/Round Timer with Client-Side Prediction
+
+Games commonly have timers to display in the UI such as Start Timer, Round Timer, and Cooldowns. Invaders also has a shared timer to ensure all players start the game at the same time. Otherwise, players with higher end devices and better network access may have an unfair advantage by loading scenes and maps faster.
```

Comment:
```suggestion
Games commonly have timers to display in the UI such as Start Timer, Round Timer, and Cooldowns. Invaders also have a shared timer to ensure all players start the game at the same time. Otherwise, players with higher-end devices and better network access may have an unfair advantage by loading scenes and maps faster.
```